### PR TITLE
first draft of GraphQL

### PR DESCRIPTION
**Pull request by marktani**
Hey, as discussed with @schickling, I'd like to contribute this blog post for GraphQL.

Currently I am not 100% finished, but will add additional material today and tomorrow :)

### CODE REVIEW

**Comment by shekhargulati** [comment]
@marktani no issues. I will review and publish once you are done.

**Comment by marktani** [comment]
I added the last few things, should be ready now :tada: Let me know what you think! :slightly_smiling_face:

File: 43-graphql/README.md [hunk 1]
```diff
@@ -0,0 +1,444 @@
...
+          "name": "Ash Ketchum"
+        }
+      }
+    ]
+  }
+}
+```
+</details>
+
+Just like with normal fields, we can even select related nodes in our query. This dramatically decreases the amount of queries needed and allows for quick prototyping and development of frontend applications. Instead of the time-draining skimming through dozens of potential endpoints, you can simply explore the possible queries interactively which has a great effect on developer experience.
```
Comment (ljacqu) [inline 43-graphql/README.md]: Minor: "This dramatically decreases the _amount_ of queries needed" should be "This dramatically decreases the _number_ of queries needed"

Comment (marktani) [reply 43-graphql/README.md]: done, thanks :)

**Review by shekhargulati — APPROVED** [review]

**Comment by shekhargulati** [comment]
Great post @marktani . Thanks for your contribution.

**Pull request merged**